skt/kernelbuilder.py

```diff
@@ -102,8 +103,12 @@ def __prepare_kernel_config(self):
         if not self.enable_debuginfo:
             self.__adjust_config_option('disable', 'debug_info')
 
-        # Set the CONFIG_LOCALVERSION to '.skt'
-        self.__adjust_config_option('set-str', 'LOCALVERSION', '.skt')
+        # Set CONFIG_LOCALVERSION
+        self.__adjust_config_option(
+            'set-str',
+            'LOCALVERSION',
+            '.{}'.format(self.localversion)
```

Comment:
Please be aware, that with the code as is skt can be made to execute arbitrary code passed via the `--localversion` option, and we need to take additional measures if we want to prevent that. Perhaps we need an issue for a security review of these things.